code/game/antagonist/outsider/actor.dm

```diff
@@ -0,0 +1,63 @@
+GLOBAL_DATUM(actors, /datum/antagonist/actor)
+
+/datum/antagonist/actor
+	id = MODE_ACTOR
+	role_text = "Actor"
+	role_text_plural = "Actors"
+	welcome_text = "You are an actor. Your role is to provide an interesting roleplay scenario for the crew. You may use your scenario's guidelines, or \
+					agree with everyone else to play something different. Your creativity is the limit! Remember that you are an antagonist, and so you have \
+					all the freedoms an antagonist does."
+	faction = "actor"
+	landmark_id = "ActorSpawnLandmark"
+	flags = ANTAG_OVERRIDE_JOB | ANTAG_CLEAR_EQUIPMENT | ANTAG_CHOOSE_NAME | ANTAG_HAS_NUKE | ANTAG_SET_APPEARANCE | ANTAG_NO_FLAVORTEXT
+	hard_cap = 10
+	hard_cap_round = 15
+	initial_spawn_req = 2 // Needs to be set to 2 to allow it to show as a group antag on the lobby setup screen.
+
+	bantype = "actor"
+
+	faction_verbs = list(
+		/mob/living/carbon/human/proc/odyssey_panel
+	)
+
+/datum/antagonist/actor/New()
+	..(1)
```

Comment:
Why are we sending a single parameter up the call chain, where `/datum/antagonist/New()` isn't taking any param? And, is it a magic number?

Reply:
who knows even, some gamemodes do it for some reason LOL